apps/omg_eth/lib/eth/root_chain.ex

```diff
@@ -228,6 +248,35 @@ defmodule OMG.Eth.RootChain do
          do: {:ok, Enum.map(logs, &decode_exit_started/1)}
   end
 
+  defp get_inflight_data(hash) do
+    {:ok, eth_tx} = Ethereumex.HttpClient.eth_get_transaction_by_hash(hash)
+    <<"0x", encode_inflight::binary>> = eth_tx["input"]
+
+    ABI.decode(
+      %ABI.FunctionSelector{
+        function: "startInFlightExit",
+        types: [:bytes, :bytes, :bytes, :bytes],
+        method_id: <<132, 97, 33, 149>>
```

Comment:
there is a function which will calculate this `method_id` based on the signature, pls use it here instead of hardcoding